lib/src/main/kotlin/com/lemonappdev/konsist/api/ext/list/KoBaseProviderListExt.kt

```diff
@@ -5,9 +5,10 @@ import com.lemonappdev.konsist.api.provider.KoBaseProvider
 /**
  * Print the elements.
  *
+ * @param prefix An optional string to be printed before each element. Default is an empty string.
  * @return The original list of elements.
  */
-fun <T : KoBaseProvider> List<T>.print(): List<T> {
-    forEach { println(it.toString()) }
+fun <T : KoBaseProvider> List<T>.print(prefix: String = ""): List<T> {
+    forEach { println(prefix + it.toString()) }
```

Comment:
The prefix should be only printed once - think about it as a header of the "print" section

```
prefix:
declaration1
declaration2
...
```